dffml/skel/source/REPLACE_IMPORT_PACKAGE_NAME/misc.py

```diff
@@ -4,10 +4,11 @@
 from dffml.repo import Repo
 from dffml.source.source import BaseSourceContext, BaseSource
 from dffml.util.cli.arg import Arg
-from dffml.util.entrypoint import entrypoint
+from dffml.util.entrypoint import entry_point
```

Comment:
`entry_point` was recently renamed to `entrypoint` throughout the rest of the codebase.

```suggestion
from dffml.util.entrypoint import entrypoint
```

Reply:
Got it. Thanks @pdxjohnny.